sorting.js

```diff
@@ -1,5 +1,12 @@
 function sortingCards(cardsDeck) {
-  return cardsDeck.sort();
-
-  // I couldn't think of a way to implement in Javascript and the one I made isn't working
+  for (var i = 0; i < cardsDeck.length; i++) {
```

Comment:
Moving forward, let's try to refrain from using `var` and use `const` and `let` instead.